test/liblll/EndToEndTest.cpp

```diff
@@ -386,6 +386,30 @@ BOOST_AUTO_TEST_CASE(keccak256_32bytes)
 		fromHex("b10e2d527612073b26eecdfd717e6a320cf44b4afac2b0732d9fcbe2b7fa0cf6")));
 }
 
+BOOST_AUTO_TEST_CASE(alloc_1)
+{
+	char const* sourceCode = R"(
+		(returnlll
+			(return (- (msize) (alloc 42))))
+	)";
+	compileAndRun(sourceCode);
+	// Note that (msize) is always a multiple of 32.
+	BOOST_CHECK(callFallback() == encodeArgs(u256(64)));
+}
+
+BOOST_AUTO_TEST_CASE(alloc_2)
+{
+	char const* sourceCode = R"(
+		(returnlll
+			(seq
+				(mstore 0x40 0xff)
+				(mstore8 (alloc 1) 42)
```

Comment:
I think this here should assign the value of `(alloc 1)` to a variable and use that in return as the starting address.

it would be nice to have another test which ensures the starting address equals to what it should be. For that maybe the test you have is enough.